src/prob/pf.jl

```diff
@@ -366,10 +366,14 @@ end
 Computes native power flow and outputs the result dict.
 """
 function compute_pf(data_math::Dict{String, Any}; v_start::Union{Dict{<:Any,<:Any},Missing}=missing, explicit_neutral::Bool=false, max_iter::Int=100, stat_tol::Float64=1E-8, verbose::Bool=false)
+    br_size = size(data_math["branch"]["1"]["br_r"],1)
+    @assert br_size <= 4  "Line impedance matrix should be up to 4x4, but is $(br_size)x$(br_size)"
```

Comment:
make this a warning not an assert, but check it for all branches - do not hard code it for the first one